hatchery/hatchery.go

```diff
@@ -704,6 +723,9 @@ func getMountFileContents(fileId string, userName string) (string, error) {
 			Config.Logger.Printf("unable to generate Nextflow config: %v", err)
 		}
 		return out, nil
+	} else if fileId == "stata.lic" {
```

Comment:
instead of hard-coding `stata.lic` here, can we make it as a value (or an array of values, if needed) from the config of `license.file-path`? 
So the idea is we should not expose and of the license files via this endpoint